IBPSA/Fluid/Actuators/Dampers/MixingBox.mo

```diff
@@ -12,10 +12,11 @@ model MixingBox "Outside air mixing box with interlocked air dampers"
     "= false to simplify equations, assuming, but not enforcing, no flow reversal"
     annotation(Dialog(tab="Assumptions"), Evaluate=true);
 
-  VAVBoxExponential damOA(
+  IBPSA.Fluid.Actuators.Dampers.Exponential damOA(
```

Comment:
- [ ] Propagate all parameters as `final`

Reply:
Done. I also restructured the code to declare parameters and IO connectors first.